src/stores/theme/createThemeStore.tsx

```
@@ -1,67 +1,263 @@
+        partialize: (state: ThemeState): PersistedThemeState => ({
+          configTheme: state.configTheme,
+          widgetTheme: state.widgetTheme,
+          configThemeStates: state.configThemeStates,
+        }),
+        onRehydrateStorage: () => {
+          return (state) => {
+            if (!state) {
+              return;
+            }
+
+            state.configTheme = state.configTheme;
```

Biome (2.1.2)

[error] configTheme is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)